documentation/book/assembly-kafka-connect.adoc

```diff
@@ -9,15 +9,13 @@
 [id='kafka-connect-{context}']
 = Kafka Connect
 
-The Cluster Operator deploys a link:https://kafka.apache.org/documentation/#connect[Kafka Connect^] cluster, which can be used with your Kafka broker deployment.
-It is implemented as a `Deployment` with a configurable number of workers.
-The default image currently contains only the `FileStreamSinkConnector` and `FileStreamSourceConnector` connectors.
-The REST interface for managing the Kafka Connect cluster is exposed internally within the {ProductPlatformName} cluster as a `kafka-connect` service on port `8083`.
-
-Example `KafkaConnect` resources and the details about the `KafkaConnect` format for deploying Kafka Connect can be found in
-xref:assembly-deployment-configuration-kafka-connect-str[Kafka Connect cluster configuration]
-and
-xref:assembly-deployment-configuration-kafka-connect-s2i-str[Kafka Connect cluster with Source2Image support].
+link:https://kafka.apache.org/documentation/#connect[Kafka Connect^], a core component of Apache Kafka, is a tool for streaming high volumes of data into and out of your Kafka cluster. It provides a framework for building connectors (a type of plug-in) for your Kafka cluster, a REST API for managing connectors, and more. Kafka Connect includes two built-in connectors, which are both available for use in {ProductName}:
```

Comment:
you can paraphrase better than me but what about "...two built-in connectors for moving files content in and out a of the Kafka cluster..."

Reply:
The connectors are tough to explain, even though I used them in my Kafka Essentials training. Does this make sense?

These connectors transfer data into and out of a Kafka cluster by using external files for sending and receiving messages.

Reply:
With this sentence it seems to me that files are used as a "tool" for transferring messages between Kafka and a different other system. Instead, you are getting data from file to kafka (source) or putting data from Kafka to a file (sink).

Reply:
> Kafka Connect includes the following built-in connectors for moving file-based data into and out of your Kafka cluster.

(Table)